### DISCUSSION TITLE
Do not index my documentation on search engines (robot.txt)

### DISCUSSION THREAD

**Discussion by PaulCollas** [requarks/wiki#6638] [Help / Questions] [answered]
In my company, I've created documentation via wiki.js. 

I only give access to the documentation via a link so that it remains "private". With this in mind, I'd like it not to appear on search engines like Google or Firefox. 

Is it possible to include a robot.txt (or some other way) so as not to be visible on search ?

In fact in Admin > General I already check this box "MetaRobots" : "No Index, No Follow"

Thanks for your answers

**Comment by NGPixel** [comment, marked as answer]
That's already what the No Index, No Follow option does...

However, that doesn't make your wiki private in any way. Search engines that don't respect robots.txt can still access your wiki and users that know your domain can still view it.

If you actually want to protect your wiki, then you need to use proper authentication and restrict access to the Guests group.

**Reply by PaulCollas** [reply]
Thank you for your reply. 

I did set up the role and access system and I did set up the system on my meta-robots.

However, I don't want users to see the links for my documentation, because some pages contain confidential information. And I don't want this confidential information to appear below the link on a search engine. 

However, where is my robot.txt file ? Because normally robot.txt should be in : mywebsite.com/robot.txt
But I'v got nothing in this url. Same for the sitemap.

Do you have any other thing to force this ? Thanks

**Reply by NGPixel** [reply]
It's /robots.txt, not /robot.txt

Again, if you have confidential info, you should restrict access to the guests group. robots.txt or sitemaps are completely irrelevant here.
